pkg/kapp/resources/identified_resources_list_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+package resources
+
+import (
+	"github.com/cppforlife/go-cli-ui/ui"
+	logger2 "github.com/k14s/kapp/pkg/kapp/logger"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func TestIdentifiedResourcesListReturnsLabeledResources(t *testing.T) {
+	mockedResTypes := new(ResourceTypesMock)
+	mockedRess := new(ResourcesMock)
+
+	mockedRess.Mock.On("All").Return(makeNewResources(t), nil)
+
+	ui := ui.NewNoopUI()
+	logger := logger2.NewUILogger(ui)
+	identifiedResources := NewIdentifiedResources(nil, mockedResTypes, mockedRess, []string{}, logger)
+
+	kappLabel := make(map[string]string)
+	kappLabel["kapp.k14s.io/app"] = "app-name"
+	sel := labels.Set(kappLabel).AsSelector()
```

Comment:
you can write these 3 lines as:

```
sel := labels.Set(map[string]string{"kapp.k14s.io/app": "app-name"}).AsSelector()
```

this gets rid of the noise of constructing objects and gets to the meat of what we are trying to do.

i would also change the label to something else so that it's not implying that this is a special label.

Reply:
Changed to a generic label: `"some-label": "value"`. Hopefully this does not imply anything special about the label.